platform/ui/src/components/InputMultiSelect/InputMultiSelect.tsx

```diff
@@ -30,7 +30,7 @@ const InputMultiSelect = ({
         value={value}
         isMulti={true}
         isClearable={false}
-        isSearchable={false}
+        isSearchable={true}
```

Comment:
I think our UI for having searchable is aweful, do you have screenshot to show before and after this change? we should move to the comboBox in the shadcn

Reply:
It doesn’t look bad to me, I formatted it well, I will post screenshots tomorrow